src/components/toast/toast.scss

```diff
@@ -173,6 +173,15 @@ md-toast {
       top: 0;
     }
 
+    // Support for RTL alignment
+    &._md-start {
```

Comment:
There're already registered through the CSS.
The toast interim automatically applies the prefix and uses the given css classes.

See the mapping line.
```js
element.addClass(isSmScreen ? '_md-bottom' : options.position.split(' ').map(function(pos) {
  return '_md-' + pos;
}).join(' '));
```
[more code](https://github.com/DevVersion/material/blob/feat/toast-rtl-positions/src/components/toast/toast.js#L389)